shanghai/main.py

```diff
@@ -14,48 +14,65 @@
 
 def exception_handler(loop, context):  # pylint: disable=unused-argument
     f = io.StringIO()
-    print("exception_handler context:", file=f)
+    print("=== Unhandled Exception ===", file=f)
+    print("- Context -", file=f)
     pprint(context, stream=f)
-    traceback.print_exc(file=f)
```

Comment:
Also, if `context['future']` is of type `asyncio.futurers.Future`, I don't think this will work. It doesn't appear the Future class has `print_stack` defined, but it's instead on `asyncio.tasks.Task`.